template/sample/check/CheckTesterProcessor.gen.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 Istio Authors.
```

Comment:
@ZackButcher can you please elaborate, after the vacation I think my brain need more context :-)